modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -170,7 +170,6 @@ Non-terminated Pods:                      (13 in total)  <10>
   openshift-machine-config-operator       machine-config-daemon-cvqw9            20m (1%)      0 (0%)      50Mi (0%)        0 (0%)
   openshift-marketplace                   community-operators-f67fh              0 (0%)        0 (0%)      0 (0%)           0 (0%)
   openshift-monitoring                    alertmanager-main-0                    50m (3%)      50m (3%)    210Mi (2%)       10Mi (0%)
-  openshift-monitoring                    grafana-78765ddcc7-hnjmm               100m (6%)     200m (13%)  100Mi (1%)       200Mi (2%)
```

Comment:
since we removed grafana pod, we also need to update
`Non-terminated Pods:                      (13 in total)`
to 
`Non-terminated Pods:                      (12 in total)`

Reply:
Good catch. Fixed.